x-pack/plugins/lens/public/editor_frame_service/service.tsx

```diff
@@ -25,10 +25,8 @@ import { Document } from '../persistence/saved_object_store';
 import { mergeTables } from './merge_tables';
 import { formatColumn } from './format_column';
 import { EmbeddableFactory, LensEmbeddableStartServices } from './embeddable/embeddable_factory';
-import { getActiveDatasourceIdFromDoc } from './editor_frame/state_management';
```

Comment:
Load some helpers async when needed because they pull in a tail of other modules